kolibri/plugins/device/assets/src/views/SelectContentPage/ChannelContentsSummary.vue

```diff
@@ -13,11 +13,12 @@
         </KTooltip>
         <h1>
           <KLabeledIcon icon="channel" :label="channel.name" />
-          <KIcon
+          <!-- Commenting out this icon because we've lost the channel.public attribute -->
+          <!-- <KIcon
             ref="lockicon"
             class="lock-icon"
             icon="unlistedchannel"
-          />
+          /> -->
```

Comment:
@bjester do you have any info here? I think this implementation may have changed recently?

Reply:
I'm not clear what this is. My work with public channel status was only in the backend.

Reply:
The `public` field is only returned on the PublicChannelSerializer. The other ChannelSerializer do not seem to have the field as part of their output.

Reply:
It's actually this change that removed the conditional too from this icon. https://github.com/jonboiser/kolibri/commit/4d9aa48f3a94d00acefa827e8fd2073bbd646d00#diff-9127258abd7abf8268a08ff621f35266

To show the icon correctly, we need to add the 'public' field to the Channel-list endpoint and bring back the v-if on the icon as well.

Reply:
I'm noticing that in my content_channelmetadata table, all of the Channels have their `public` field equal to null. This feature is new to me, so I'm assuming at some point at import this field was supposed to be set to True or False?

Reply:
On the Kolibri side, the `public` field is brand new. I believe that information was being used in a response from studio. If it's needed, I would suggest adding it.

Reply:
Yeah it's `null` for channels imported prior to the migration since that's an annotation at time of import.

Reply:
Oh yeah I'm seeing this, but it seems to label USB-imported channels as public = null by default.